internal/uploader/events_uploader.go

```diff
@@ -173,6 +171,16 @@ func prepareFiles(ctx context.Context, db *gorm.DB, cluster *common.Cluster, eve
 	return buffer, nil
 }
 
+func metadataFile(tw *tar.Writer, clusterID *strfmt.UUID, config Config) {
+	metadata := versions.GetModelVersions(config.Versions)
+	metadata["deployment-type"] = config.DeploymentType
+	metadata["deployment-version"] = config.DeploymentVersion
+
+	if metadataJson, err := json.Marshal(metadata); err == nil {
+		addFile(tw, metadataJson, fmt.Sprintf("%s/metadata.json", *clusterID)) //nolint:errcheck // errors adding this file shouldn't prevent the data from being sent
```

Comment:
I know that we briefly discussed this, but I now realize that this would break the consumer. If necessary we can change the consumer first to handle both cases, but I think it'd be best to keep compatibility and just add metadata.json and versions.json separately. WDYT?

Reply:
Oh gotcha, sure a separate file will work! I'll modify this to reflect it momentarily

Reply:
I kept the version in the metadata file too so that when the versions.json file is removed, it'll already be part of the metadata.json file.